ext/search_kit/Civi/Api4/Action/SearchDisplay/AbstractRunAction.php

```diff
@@ -574,6 +574,15 @@ private function checkLinkCondition(array $item, array $data): bool {
       }
       return TRUE;
     }
+    // Convert the conditional value of 'current_domain' into an actual value that filterCompare can work with
+    if (substr($item['condition'][0], 0, 9) === 'domain_id' && $item['condition'][2] === 'current_domain') {
```

Comment:
This bit (`substr($item['condition'][0], 0, 9) === 'domain_id'`) looks like stringey-guesswork. I don't think it would work with joins, or with any field that doesn't happen to be named `domain_id`. IMO just omit it.

Reply:
yeh I think i was just trying to limit when this could be done but your comment makes sense updated